playground/mocks/idp/idx/data/identify.json

```diff
@@ -20,6 +20,11 @@
                   "name":"identifier",
                   "label":"Username"
               },
+              {
+               "name":"rememberMe",
```

Comment:
just double check, Is `rememberMe` field always in the API?

Reply:
I believe so ( https://github.com/okta/okta-core/pull/42373 ), @jadsiblini-okta can confirm it certainly.

Reply:
@jadsiblini-okta you mean defaults to `"visible" : "false"` ?

Reply:
I believe @jadsiblini-okta is talking about the POST response from UI from identify step. If `rememberMe` is not sent, it will be considered `false` gracefully.

Reply:
Right